xchangelog

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# xchangelog PKGNAME - open package changelog
+
+XBPS_DISTDIR=$(xdistdir) || exit 1
+
+pkg=$1
+[ -z "$pkg" ] && pkg=$(basename "$PWD")
+
+if [ -f "$pkg" ]; then
+	template="$pkg"
+elif [ -f "$pkg/template" ]; then
+	template="$pkg/template"
+elif [ -f "$XBPS_DISTDIR/srcpkgs/$pkg/template" ]; then
+	template="$XBPS_DISTDIR/srcpkgs/$pkg/template"
+else
+	exit 1
+fi
+
+. "$template"
+
+if [ -z "$changelog" ]; then
+	echo "No changelog defined in template"
+	exit 1
+fi
+
+if curl -sI -- "$changelog" | grep -i "^content-type: text/plain"; then
```

Comment:
```suggestion
if curl -sI -- "$changelog" | grep -i "^content-type: text/"; then
```

Reply:
That won't work because HTML's MIME type is `text/html`.